components/feature/download/README.md

```diff
@@ -0,0 +1,19 @@
+# [Android Components](../../../README.md) > Feature > Download
+
+Feature implementation for apps that want to use Android downloads manager.
```

Comment:
Let's link to the Android docs: [Android's Download Manager](https://developer.android.com/reference/android/app/DownloadManager)

Reply:
Should I also update the diagram in the main README.md?
I noticed that Intent and Tabs features aren't there too.

![image](https://user-images.githubusercontent.com/773158/46738997-8c2c1180-cc6d-11e8-99c5-f0c7eadf224e.png)

Reply:
👍